src/patternfly/sass-utilities/mixins.scss

```diff
@@ -29,13 +29,39 @@
   }
 }
 
-// Creates flex item
-@mixin pf-utility-builder($class-name) {
-  // generate column span modifiers
-  @each $breakpoint, $breakpoint-value in $pf-global--breakpoint-map {
-    .pf-u-#{$class-name}#{$breakpoint-value} {
-      @include pf-media-query($breakpoint) {
-        @content;
+/// Create single prop / value classes, optionally add responsive suffix
+/// @group mixins
+/// @moduleType mixin
+/// @parameter: {Suffix} xs, sm, md, lg, xl, base or null
+/// @usage: @include pf-utility-builder(class-name, base sm md lg xl);
+/// ===============================================================================================
+/// non-responsive, base only                @include pf-utility-builder(class-name) { prop: val !important; }
+/// responsive, not including base           @include pf-utility-builder(class-name, sm lg) { prop: val !important; }
+/// responsive, including base               @include pf-utility-builder(class-name, base sm lg) { prop: val !important; }
+/// responsive, including all breakpoints    @include pf-utility-builder(class-name, $pf-global--breakpoint-list) { prop: val !important; }
+/// ===============================================================================================
+@mixin pf-utility-builder($class-name, $breakpoints: null) {
+  // if only class-name is passed to mixin, with no breakpoint params, return the prop / value declaration
+  @if $breakpoints == null or $breakpoints == "base" {
+    .pf-u-#{$class-name} {
+      @content;
+    }
+  }
+
+  // if class-name and responsive list values are passed to mixin
+  @else {
+    @each $breakpoint in $breakpoints {
+      $suffix: -on-#{$breakpoint};
+
+      // if no media query, set suffix to null
+      @if $breakpoint == null or $breakpoint == "base" {
```

Comment:
What's the benefit of having `null` and `base`? Couldn't you just set the default in this mixin to `base` (instead of `null`) and in `$pf-global--breakpoint-list` change `null` to `base`?

Reply:
Yep! Nice!